index.html

```diff
@@ -158,8 +158,8 @@ <h3 id="server-performance"><a href="#server-performance">Server performance</a>
             <td class="ok"><a href="http://httpd.apache.org/docs/trunk/mod/mod_ssl.html#sslalpnpreference">yes</a></td>
             <td class="ok"><a href="https://wiki.mozilla.org/Security/Server_Side_TLS#Apache">yes</a></td>
             <td class="ok"><a href="https://httpd.apache.org/docs/trunk/mod/mod_http2.html">yes</a></td>
-            <td class="alert">no</td>
-            <td class="alert">no</td>
+            <td class="ok"><a href="https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslciphersuite">yes</a></td>
+            <td class="ok">yes (if using OpenSSL)</td>
```

Comment:
Any docs for the OpenSSL + 0RTT bit that we can point to?

We don't need the comment, we can just link "yes".

Reply:
There is no link in the apache docs for 0-rtt. I can change it just to yes.